bip-p2qrh.mediawiki

```diff
@@ -140,6 +149,4 @@ To help implementors understand updates to this BIP, we keep a list of substanti
 
 == Acknowledgements ==
 
-TBD
-
-Much gratitude to my co-founder, Kyle Crews for proofreading and editing, and also to Dave C. who provided the name, QuBit.
+Much gratitude to my co-founder, Kyle Crews for proofreading and editing, to David C. who provided the name, QuBit. Same for those who took the time to review and contribute, including AB, (and hopefully many more).
```

Comment:
nit pic on wording:

Much gratitude to my co-founder, Kyle Crews for proofreading and editing, and to David C. who suggested the name "QuBit." Thank you as well to those who took the time to review and contribute, including AB (and hopefully many more).